--- vvdec-3.1.0/cmake/modules/vvdecInstall.cmake.orig	2025-11-18 12:22:17.000000000 +0100
+++ vvdec-3.1.0/cmake/modules/vvdecInstall.cmake	2026-03-21 09:58:41.521724956 +0100
@@ -75,10 +75,14 @@ endif()
 
 
 # install targets
+if( CMAKE_BUILD_TYPE )
+install_targets( "${INSTALL_TARGETS}" "${CMAKE_BUILD_TYPE}" )
+else()
 install_targets( "${INSTALL_TARGETS}" Release )
 install_targets( "${INSTALL_TARGETS}" Debug )
 install_targets( "${INSTALL_TARGETS}" RelWithDebInfo )
 install_targets( "${INSTALL_TARGETS}" MinSizeRel )
+endif()
 
 # install pdb files
 install_lib_pdb( vvdec )
@@ -107,10 +111,15 @@ else()
 endif()
 
 # create target cmake files
+if( CMAKE_BUILD_TYPE )
+string( TOLOWER ${CMAKE_BUILD_TYPE} build_type_lc_ )
+install( EXPORT vvdecTargets-${build_type_lc_} NAMESPACE vvdec:: FILE vvdecTargets-${CONFIG_POSTFIX}.cmake CONFIGURATIONS ${CMAKE_BUILD_TYPE} DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/vvdec )
+else()
 install( EXPORT vvdecTargets-release        NAMESPACE vvdec:: FILE vvdecTargets-${CONFIG_POSTFIX}.cmake CONFIGURATIONS Release        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/vvdec )
 install( EXPORT vvdecTargets-debug          NAMESPACE vvdec:: FILE vvdecTargets-${CONFIG_POSTFIX}.cmake CONFIGURATIONS Debug          DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/vvdec )
 install( EXPORT vvdecTargets-relwithdebinfo NAMESPACE vvdec:: FILE vvdecTargets-${CONFIG_POSTFIX}.cmake CONFIGURATIONS RelWithDebInfo DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/vvdec )
 install( EXPORT vvdecTargets-minsizerel     NAMESPACE vvdec:: FILE vvdecTargets-${CONFIG_POSTFIX}.cmake CONFIGURATIONS MinSizeRel     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/vvdec )
+endif()
 
 
 function( resolve_target_interface_libs TGT OUT_VAR )
